ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarDemoController.swift

```diff
@@ -67,6 +65,36 @@ class AvatarDemoController: DemoTableViewController {
 
             return cell
 
+        case .imageBasedRingColor:
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: BooleanCell.identifier) as? BooleanCell else {
+                return UITableViewCell()
+            }
+
+            cell.setup(title: row.title, isOn: self.isSettingOn(row: row))
+            cell.titleNumberOfLines = 0
+            cell.onValueChanged = { [weak self, weak cell] in
+                self?.updateSetting(for: row, isOn: cell?.isOn ?? true)
+            }
+
+            customRingIndexPath = indexPath
+
+            return cell
+
+        case .ring:
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: BooleanCell.identifier) as? BooleanCell else {
+                return UITableViewCell()
+            }
+
+            cell.setup(title: row.title, isOn: self.isSettingOn(row: row))
+            cell.titleNumberOfLines = 0
+            cell.onValueChanged = { [weak self, weak cell] in
+                self?.updateSetting(for: row, isOn: cell?.isOn ?? true)
+            }
+
+            ringIndexPath = indexPath
```

Comment:
couldn't .imageBaseRingColor and .ring be combined except for differentiating ringIndexPath code? seems lot of duplicates